docker/startup

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+RCLONE_CONFIG=/config/rclone/rclone.conf
+
+if [ -z "$REMOTE_NAME" ]; then
+    echo "REMOTE_NAME is not set"
+    exit 1
+fi
+if [ -z "$REMOTE_URL" ]; then
+    echo "REMOTE_URL is not set"
+    exit 1
+fi
+if [ -z "$REMOTE_VENDOR" ]; then
+    echo "REMOTE_VENDOR is not set"
+    exit 1
+fi
+
+if [ "$REMOTE_NEXTCLOUD_CHUNK_SIZE" != "0" ]; then
+    REMOTE_NEXTCLOUD_CHUNK_SIZE="${REMOTE_NEXTCLOUD_CHUNK_SIZE}"i
+fi
+
+mkdir -p "$(dirname "$RCLONE_CONFIG")"
+
+cat <<EOF >"$RCLONE_CONFIG"
+[$REMOTE_NAME]
+type = webdav
+url = $REMOTE_URL
+vendor = $REMOTE_VENDOR
+nextcloud_chunk_size = ${REMOTE_NEXTCLOUD_CHUNK_SIZE}
+EOF
+
+# parse args
+ifsBak=$IFS
+IFS=' '
+
+ARGS=""
+for arg in $PROXY_ARGS; do
+    ARGS="$ARGS $arg"
+done
+
+IFS=$ifsBak
+
+rclone serve s3 "$REMOTE_NAME": "$ARGS"
```

Comment:
this does not work for me, rclone complains that I have supplied two arguments, but should only send one 
running `docker run --rm -p 8080:8080 -e REMOTE_NAME=nc -e REMOTE_URL="http://localhost/nextcloud-server/remote.php/dav/files/admin/roots3" -e REMOTE_VENDOR=nextcloud --name s3-webdav-proxy --network=host ghcr.io/jankaritech/s3-webdav-proxy`

it says: `Command s3 needs 1 arguments maximum: you provided 2 non flag arguments: ["nc:" ""]`